src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,16 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
-        return 0;
+        for (String value : records) {
+            String[] data = value.split(":");
+            if (data.length == 2 && data[0].equals(email)) {
```

Comment:
```suggestion
            if (data[0].equals(email)) {
```
we can suppose that data will have the same form